js/interface.js

```diff
@@ -2257,6 +2276,21 @@ $('#fl-store-2fa-select, #fl-ent-2fa-select').on('change', function (e) {
   socket.to(socketClientId).emit('aab.apple.login.2fa.device', e.target.value);
 });
 
+Fliplet().then(function () {
+  checkDemoUser();
+});
+
+// This listener need that we can understand that it is a user entering data but not an any password managers.
+$('#fl-store-revDemoUser').on('keyup', function (event) {
```

Comment:
User input could come from typing (`keyup`), pasting using Ctrl/cmd + V (`keyup`), pasting using right-click menu or browser menu (Edit > Paste). I therefore don't think we can reliably use `keyup` here to determine if user has input anything.